src/components/Modal/BaseModal.js

```diff
@@ -90,8 +92,9 @@ function BaseModal({
 
             // To prevent closing any modal already unmounted when this modal still remains as visible state
             Modal.setCloseModal(null);
-        };
-    }, [hideModal, isVisible, onClose]);
+        },
+        [hideModal, isVisible],
```

Comment:
We can remove these dependencies as not required for cleanup.

Reply:
yep I believe that's what I have done. @Pujan92 could you check the latest code?

Reply:
I am still seeing the dependencies, hope you have committed. Can you plz check once

Reply:
ah I need to move all dependencies? didn't know that fixed!